packages/wonder-blocks-modal/src/components/one-pane-dialog.tsx

```diff
@@ -74,7 +74,7 @@ type Common = {
     /**
      * The ID of the content describing this dialog, if applicable.
      */
-    ["aria-describedby"]?: string;
```

Comment:
@jeresig this was causing the properties to change from optional to required by `flowgen`.  This fixes the issue.